modules/mod_authentication/templates/_logon_reset_form_fields.tpl

```diff
@@ -20,7 +20,7 @@
 %}
 <div class="form-group">
     <label class="control-label" for="password_reset1">{_ New password _}</label>
-    <input class="form-control" type="password" id="password_reset1" name="password_reset1" value="" autocomplete="new-password" placeholder="{_ New password _}" />
+    <input class="form-control {% if not is_password_change %}do_autofocus{% endif %}" type="password" id="password_reset1" name="password_reset1" value="" autocomplete="new-password" placeholder="{_ New password _}" />
```

Comment:
Shouldn't it autofocus if is_password_change is true, instead the other way round?

Reply:
If password change is true then there is another field to enter the current password, that field gets then the focus.